Makefile

```diff
@@ -37,7 +37,6 @@ lint: ## Run the linter
 	$(info Running linting...)
 	flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
 	flake8 . --count --max-complexity=10 --max-line-length=127 --statistics
-	pylint service
```

Comment:
What was the reason to add this line in the first place?

Reply:
This line is included in lab_bluemix, but I guess we don't need it.